src/__tests__/linter.test.ts

```diff
@@ -144,6 +142,8 @@ describe('linter', () => {
   });
 
   test('should not report anything for disabled rules', async () => {
+    spectral.registerFormat('oas2', isOpenApiv2);
```

Comment:
Can you add a context and best then inside it?

describe(“when os2 and oas3 are registered as formats”

Then all tests which need that go inside